cpptod.dd

```diff
@@ -800,7 +809,7 @@ template IsFunctionT(T)
 
 void test()
 {
-    typedef int fp(int);
+    alias int fp(int);
```

Comment:
I think I've run into this issue before. If it is what I think it is, it's not possible to translate this to the new alias syntax.